tests/integrations/resource_metering/test_read_keys.rs

```diff
@@ -0,0 +1,250 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
This file includes integration tests for scan row (including PointGet/Scan/Coprocessor). cc @breeswish